src/art_net/4C_art_net_impl_stationary.cpp

```diff
@@ -460,6 +473,42 @@ void Arteries::ArtNetImplStationary::prepare_time_loop()
   return;
 }
 
+/*----------------------------------------------------------------------*
+ *----------------------------------------------------------------------*/
+void Arteries::ArtNetImplStationary::collect_runtime_output_data()
+{
+  // write domain decomposition for visualization (only once at step 0!)
+  if (step_ == upres_ or step_ == 0) visualization_writer_->append_element_owner("Owner");
```

Comment:
I have seen that it already used to be like this. However, if the comment should be true, you need to change as follows:
```suggestion
  if (step_ == 0) visualization_writer_->append_element_owner("Owner");
```